charts/newrelic-prometheus-agent/templates/_helpers.tpl

```diff
@@ -53,6 +48,38 @@ common:
 
 {{- end -}}
 
+{{- /* it builds the extra_write_relabel_configs configuration merging: lowdatamode, user ones, and metrictyperelabeldefaults  */ -}}
+{{- define "newrelic-prometheus.configurator.extra_write_relabel_configs" -}}
+
+{{- $extra_write_relabel_configs := list  -}}
+{{- if .Values.config.metric_type_override -}}
+  {{- if .Values.config.metric_type_override.enabled -}}
+    {{- $metricTypeOverride := .Files.Get "static/metrictyperelabeldefaults.yaml" | fromYaml -}}
+    {{- $extra_write_relabel_configs = concat $extra_write_relabel_configs $metricTypeOverride.metrics_type_relabel -}}
+  {{- end -}}
+{{- end -}}
+
+
+{{- if (include "newrelic.common.lowDataMode" .) -}}
+  {{- $lowDataModeRelabelConfig := .Files.Get "static/lowdatamodedefaults.yaml" | fromYaml -}}
+  {{- $extra_write_relabel_configs = concat $extra_write_relabel_configs $lowDataModeRelabelConfig.low_data_mode -}}
```

Comment:
Just wondering if we checked for unwanted relabel filtering while having `low_data_mode` enabled relabels as we are now "prepending" first the `metrics_type_relabel`.